src/compiler/compile/render_dom/wrappers/Element/Attribute.ts

```diff
@@ -67,15 +68,27 @@ export default class AttributeWrapper extends BaseAttributeWrapper {
 			}
 		}
 
-		this.name = fix_attribute_casing(this.node.name);
-		this.metadata = this.get_metadata();
-		this.is_indirectly_bound_value = is_indirectly_bound_value(this);
-		this.property_name = this.is_indirectly_bound_value
-			? '__value'
-			: this.metadata && this.metadata.property_name;
+		if (this.parent.node.namespace && this.parent.node.namespace == external) {
```

Comment:
Wondering if I’m missing something here, but could this not be simplified to just:

`if(this.parent.node.namespace === external)`?

Or was it `this.parent.node` you’d meant to safety-check?